client/components/common/UserCustomSets.tsx

```diff
@@ -196,269 +63,11 @@ const UserCustomSets: React.FC<Props> = ({ username }) => {
             },
           }}
         >
-          <h1 css={{ margin: '10px 0px 0px 0px', fontSize: 28 }}>{`${capitalize(
-            username!,
-          )}`}</h1>
+          <h1 css={{ margin: '10px 0px 0px 0px', fontSize: 28 }}>{username}</h1>
```

Comment:
I've always tried to stick with multiples of 4 for margins, and `rem` for font sizes. Also, wouldn't `marginTop` here be sufficient?

Reply:
Changed the margin to 12px, and font size to 1.6rem. `marginTop` alone wouldn't work because the default margins on an h1 element will move the _Member since_ too low, so I explicitly had to make the other margins 0px.